aea/cli/common.py

```diff
@@ -117,13 +117,13 @@ def _try_to_load_protocols(ctx: Context):
     for protocol_name in ctx.agent_config.protocols:
         logger.debug("Processing protocol {}".format(protocol_name))
         try:
-            ctx.protocol_loader.load(open(os.path.join("protocols", protocol_name, DEFAULT_PROTOCOL_CONFIG_FILE)))
+            ctx.protocol_loader.load(open(os.path.join("protocols", protocol_name.package_name, DEFAULT_PROTOCOL_CONFIG_FILE)))
         except FileNotFoundError:
             logger.error("Protocol configuration file for protocol {} not found.".format(protocol_name))
             sys.exit(1)
 
         try:
-            protocol_spec = importlib.util.spec_from_file_location(protocol_name, os.path.join("protocols", protocol_name, "__init__.py"))
+            protocol_spec = importlib.util.spec_from_file_location(protocol_name, os.path.join("protocols", protocol_name.package_name, "__init__.py"))
```

Comment:
should it be protocol_id?

Reply:
The namings are going to change.
However, at the moment, to be precise, `protocol_name` is an instance of `ProtocolId`, and `package_name` is the name of the package, e.g. in the public id of the protocol `fetchai/oef:0.1.0`, `package_name` is `oef`.

Since we store the components in directories whose names are the names of the packages that they contain, we have to use `package_name` to point to them.

In the example, for the protocol `oef`, we have to load `protocols/oef/__init__.py`
```
protocols/
    oef/
        __init__.py
        protocol.yaml
        ...
```